app/src/main/java/com/codesoom/assignment/dto/ErrorResponse.java

```diff
@@ -1,12 +1,19 @@
 package com.codesoom.assignment.dto;
 
+/**
+ * 에러에 대한 정보를 저장하고 처리합니다.
+ */
```

Comment:
이 클래스는 에러에 대한 정보를 저장은 하지만 처리는 하고 있지 않은 것 같아요.
```suggestion
/**
 * 에러가 발생한 상황을 보고합니다.
 */
```
```suggestion
/**
 * 에러 상황의 정보를 표현합니다.
 */
```